bug fixes and add new features

bu11ettrain opened the pull request:
- add extra config args handling (to specify gpus and etc)
- add "Pool URL" handling (to specify more than 1 pool)
- add accepted shares counter
- add nvtool OC adjusting from extra config args
- fix miner uptime
- fix gpu order and stats when gpus specified partly
- fix logs path
- fix RAM \ hdd space overflow due to error "Failed find valid proof target in range" and add logging busid for faulted GPUs

bu11ettrain committed:
bug fixes and add new features

- add extra config args handling (to specify gpus and etc)
- add "Pool URL" handling (to specify more than 1 pool)
- add accepted shares counter
- add nvtool OC adjusting from extra config args
- fix miner uptime
- fix gpu order and stats when gpus specified partly
- fix logs path
- fix RAM \ hdd space overflow due to error "Failed find valid proof target in range" and add logging busid for faulted GPUs

mumu714 commented:
Hi @bu11ettrain, thanks for your contribution.
Two things that need to be modified: 
1. no need "Pool URL" (Just specify three mining pools in the h-run.sh)
2. extra config args need to support specifying custom_name

bu11ettrain committed:
fixes

remove Pool URL parsing
add --custom_name applying from extra config args

bu11ettrain committed:
fixes

remove Pool URL parsing (pool are now hardcoded)
add --custom_name applying from extra config args

mumu714 commented:
Hi @bu11ettrain, thanks for your contribution.
Please leave your Discord Handler. We will contact you and send the reward.

bu11ettrain commented:
User ID: 367200150944481290
@dmppmd
